resolwe_bio/processes/reads_processing/bbduk.yml

```diff
@@ -479,6 +496,9 @@
           qtrim={{ operations.quality_trim }} \
         {% endif %} \
         trimq={{ operations.trim_quality }} \
+        {% if operations.ignore_bad_quality %} \
```

Comment:
`qin={{ operations.quality_encoding_offset }} \` is missing for the ``bbduk-single`` process.

Reply:
Nice catch, thanks!